butler-bot/bot.js

```diff
@@ -67,6 +78,46 @@ client.on('message', (message) => {
     .then(() => { logger.info('Successfully added the role') })
 })
 
+// Kick users who haven't introduced themselves within two weeks
+client.setInterval((servers) => {
+  const itMuslimServer = servers.get(IT_MUSLIM_SERVER_ID)
+
+  const restrictedMembers = itMuslimServer.members.filter((member) => {
+    return !member.roles.has(FULL_MEMBER_ROLE_ID)
+  })
+
+  // Today's date
+  let now = new Date()
+
+  restrictedMembers.forEach((member) => {
+    if (member.user.bot) {
+      return
+    }
+
+    // Сalculation of the time interval in which the user was inactive
+    let daysInactive = (now.getTime() - member.joinedTimestamp) / MILLISECONDS_IN_DAY
+    if (daysInactive > 14) {
+      member.kick()
+        .then(() => console.log(`Kicked ${member.displayName}`))
+        .catch(console.error)
+    }
+
+    // Сalculation of the hours in which the user was inactive
+    let hoursInactive = Math.floor((now.getTime() - member.joinedTimestamp) / 3600000)
```

Comment:
It should work without `Math.floor`. What is `3600000`? Can we change it by some constant? Is it `MILLISECONDS_IN_HOUR`?